tensorflow/contrib/rnn/BUILD

```diff
@@ -27,6 +52,17 @@ cuda_py_tests(
     ],
 )
 
+cuda_py_tests(
```

Comment:
I think you might be able to just add your srcs to line 47 above, since it's using a cuda_py_tests (note the plural)